frontmatter/titlepage.tex

```diff
@@ -77,8 +77,9 @@
 	\end{figure}	\vspace{5mm}
 
 	Department of Computer Science and Engineering\\
+        \ifdefined\thedivision
```

Comment:
Is this indentation intended?

Reply:
Huh no, it is not... Good catch!

Reply:
Ugh, it's a tabs-vs-spaces thing...
For some reason my editor does not respect the original tabs and instead indents it with spaces..

Reply:
What? Are these files indented with tabs :angry:? I've never noticed that. I hate tabs and propose that we get rid of all of them.